hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenAttributesValidator.java

```diff
@@ -57,6 +64,36 @@ public TokenAttributesValidator() {
         // Dagger
     }
 
+    /**
+     * Performs a "pure" check for a given account ID. If the account ID is not in the body, is null, or is invalid, a {@link PreCheckException} is thrown.
+     */
+    public static void validateAccountId(final boolean bodyHasAccountId, @Nullable final AccountID accountID)
+            throws PreCheckException {
+        validateTruePreCheck(bodyHasAccountId, INVALID_ACCOUNT_ID);
+        validateAccountID(accountID, INVALID_ACCOUNT_ID);
+    }
+
+    /**
+     * Performs a "pure" check for a given token ID. If the token ID is not in the body, is null, or is invalid, a {@link PreCheckException} is thrown.
+     */
+    public static void validateTokenId(final boolean bodyHasTokenId, @Nullable final TokenID tokenID)
+            throws PreCheckException {
+        validateTruePreCheck(bodyHasTokenId, INVALID_TOKEN_ID);
+        validateTruePreCheck(tokenID != null, INVALID_TOKEN_ID);
```

Comment:
Why do we need this check again ? `bodyHasTokenID` means it is not null right ?

Reply:
ohhh I was thinking `hasTokenId` checked if it was the default token. I get really confused with the default token and checking for null..

Reply:
I removed the extra check